bin/os_main.tf

```diff
@@ -47,45 +47,42 @@ resource "openstack_compute_instance_v2" "kubenow-image-export" {
   flavor_name     = "8C-8GB-50GB"
   key_pair        = "${openstack_compute_keypair_v2.main.name}"
   security_groups = ["default"]
-  floating_ip     = "${openstack_networking_floatingip_v2.fip_1.address}"
 
   network {
     uuid = "${var.network_id}"
   }
+}
+
+# Provisioners: copying files so to be executed remotely on the VM instance
+resource null_resource "upload-and-execute-file" {
+  triggers {
+    instance_id = "${openstack_compute_instance_v2.kubenow-image-export.id}"
+  }
+
+  connection {
+    type        = "ssh"
+    host        = "${openstack_networking_floatingip_v2.fip_1.address}"
+    user        = "${var.ssh_user}"
+    private_key = "${file(var.ssh_key_prv)}"
+    agent       = "false"
+    timeout     = "3m"
+  }
```

Comment:
Currently this seems to be a workaround given the issue we've filed with the Terraform's folks: https://github.com/hashicorp/terraform/issues/19565. So we are provisioning the necessary files via an "`upload-and-execute-file`" resource.